src/components/signoff/SignoffToolBar.tsx

```diff
@@ -79,6 +79,10 @@ export default function SignoffToolBar({
     }
   }, [collectionState.data]);
 
+  if (!bucketState.data?.id) {
```

Comment:
When does this happen exactly? Or in other words, why navigating to a read-only bucket leads to bucket data being empty?

If I navigate to `/buckets/bid/collections/cid`, even if I don't have permissions to see the bucket details, the bucket data could at least be `{"data": {"id": "bid"}}`

Reply:
Replied in other comment, but this only happens when a user can view record data but not the collection attributes.

Reply:
Ok, I see. So this patch hides the signoff toolbar if no permissions. Makes sense.